javascript/index.js

```diff
@@ -0,0 +1,18 @@
+const algoliasearch = require('algoliasearch');
+
+var applicationId = '';
```

Comment:
Nit: Could we handle those kind of variables with env variable? IIRC we can have a `.env` file and access it in Node with something like this `process.env.APP_ENV`.  Also, we need to add `.env` files to the `.gitignore` to prevent users from pushing their secrets.